{"name": "Принятие заказа с различными невалидными ID курьеров", "status": "passed", "steps": [{"name": "Попытаться принять заказ с невалидным ID курьера: 9999999999", "status": "passed", "steps": [{"name": "Принять заказ 546575 курьером '9999999999'", "status": "passed", "parameters": [{"name": "order_id", "value": "546575"}, {"name": "courier_id", "value": "'9999999999'"}], "start": 1758527546085, "stop": 1758527546423}], "start": 1758527546085, "stop": 1758527546423}, {"name": "Проверить что запрос завершился ошибкой", "status": "passed", "start": 1758527546423, "stop": 1758527546423}], "parameters": [{"name": "invalid_courier_id", "value": "'9999999999'"}], "start": 1758527546085, "stop": 1758527546425, "uuid": "6d5087ac-3d99-4b3e-bb8d-8bf756e4bc9f", "historyId": "dc10bb2b475f3d8f32141fddc32a8ab5", "testCaseId": "4409f057984de48866aed05d3c5372e9", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_various_invalid_courier_ids", "labels": [{"name": "epic", "value": "Order Management"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Accept Order API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}